addons/sys_list/stringtable.xml

```diff
@@ -41,5 +41,26 @@
             <German>Zeigt Benachrichtigungen für Funk und Funkgeräte.</German>
             <Japanese>無線機と送信に関する通知を表示します。</Japanese>
         </Key>
+        <Key ID="STR_ACRE_sys_list_hintBgColor_displayName">
+            <English>Hint Background Color</English>
+        </Key>
+        <Key ID="STR_ACRE_sys_list_broadcastColor_displayName">
+            <English>Transmitting Color</English>
```

Comment:
String name uses "Broadcast", actual string uses "Transmitting". I would use "Transmission" in both since it's a new string.